src/wireviz/DataClasses.py

```diff
@@ -1,32 +1,41 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
-from typing import Optional, List, Any, Union
+from typing import Optional, List, Union
 from dataclasses import dataclass, field
 from wireviz.wv_helper import int2tuple
 from wireviz import wv_colors
 
 
+Name = str # Case insensitive unique name of connector or cable
+Pin = Union[int, str] # Pin identifier
+Wire = Union[int, str] # Wire number or 's' for shield
+MLstr = str # Multi-line string where any newline is properly handled
```

Comment:
If PR #164 is accepted, then we might need 3 different text types:
- Text = Text that might contain HTML hyperlinks that are removed except in HTML output.
- TextML = Text that might contain HTML hyperlinks that are removed except in HTML output, and newlines that are translated to `<br/>` in diagram output or to space otherwise.
- str = Plain text that doesn't contain any HTML tags nor newline.